src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -617,3 +617,19 @@ message BlobWriteKey {
 
   string identifier = 2;
 }
+
+message MemoryUsage {
```

Comment:
This seems vestigial of a general purpose memory usage behavior - prefer an explicit message definition here that contains the counts involved (entry and directories). I'd also like to add the total size of the `containingDirectories`, which should be aggregated and retained with total and max counts.